benchmark/README.md

```diff
@@ -60,9 +60,20 @@ TPCH timing results is written to stdout in the following form: `TPCH_Result,<la
     <true for sql tests, false for functional tests>
     ```
 
+    **Note**: Ensure that you build the worker and application with .NET Core 3.0 in order to run hardware acceleration queries.
+
+
 ## Python
 1. Upload [run_python_benchmark.sh](run_python_benchmark.sh) and all [python tpch benchmark](python/) files to the cluster.
-2. Run the benchmark by invoking:
+2. Install pyarrow and pandas on the cluster including head and worker nodes. You can create a bash script with the following commands and run that bash script on all the nodes in the cluster (Following instructions use Conda to install packages, you can also use other package management softwares such as pip).

```

Comment:
Install pyarrow and pandas on all nodes in the cluster. For example, if you are using Conda, you can use the following commands to install them.

Reply:
Made the change, thanks!